mailpoet/lib/WooCommerce/MailPoetTask.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\WooCommerce;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+use MailPoet\Config\Menu;
+use MailPoet\DI\ContainerWrapper;
+use MailPoet\Settings\SettingsController;
+
+/**
+ * MailPoet task that is added to the WooCommerce homepage.
+ */
+class MailPoetTask extends Task {
+  /**
+   * @return string
+   */
+  public function get_id() { // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```

Comment:
It is just a detail, but could we use a return type hint instead of the PHP doc return parameter? Tyhehints are checked runtime PHPdoc might become outdated. Or is there some reason/risk why we can't use type hints?

Reply:
Good point. I used an example class from a WooCommerce extension and didn't pause to reflect on the fact that I could use return type hints instead of docblocks precisely for the reason that you mentioned. Thanks for pointing that out. It is now fixed.